src/Plugin/Field/FieldFormatter/EmbeddableFormatter.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+namespace Drupal\stanford_media\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Url;
+use Drupal\media\Entity\MediaType;
+use Drupal\media\OEmbed\Resource;
+use Drupal\media\OEmbed\ResourceException;
+use Drupal\stanford_media\Plugin\media\Source\Embeddable;
+use Drupal\media\Plugin\Field\FieldFormatter\OEmbedFormatter;
+
+/**
+ * Field formatter for embeddables.
+ *
+ * @FieldFormatter (
```

Comment:
Is there a module that provides this at all?

Reply:
This is an extension of an existing class in core
`class EmbeddableFormatter extends OEmbedFormatter`